core/src/language/swift.rs

```diff
@@ -778,7 +781,12 @@ impl Swift {
             decs.push(CODABLE);
         }
 
-        writeln!(w, "public struct CodableVoid: {} {{}}", decs.join(", "))
+        format!("\n/// () isn't codable, so we use this instead to represent Rust's unit type\npublic struct CodableVoid: {} {{}}\n", decs.join(", "))
+    }
```

Comment:
These two snapshot tests will need to be updated for this new output:

    can_generate_generic_struct::swift
    can_handle_unit_type::swift

You can run:

```bash
UPDATE_EXPECT=1 cargo t --all-features --test snapshot_tests can_generate_generic_struct
UPDATE_EXPECT=1 cargo t --all-features --test snapshot_tests can_handle_unit_type
```